torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1489,8 +1498,7 @@ void InitXlaModuleBindings(py::module m) {
           << "Input shard shape must include padding: " << shard.sizes()
           << " vs " << shard_shape;
     }
-    auto xla_devices = GetXlaDevices(devices);
-    auto xla_data = ShardingUtil::CreateShardedData(shards, xla_devices,
+    auto xla_data = ShardingUtil::CreateShardedData(shards, devices,
```

Comment:
@alanwaketan this is where device `tpu:0` being used. it used to be `devices` is `xla:0` and we used `GetXlaDevices` to convert it to `tpu:0`. Now we passed `tpu:0` directly to `CreateShardedData`. Now we just don't call `GetXlaDevices` at all.

Reply:
Okay, then I think it's safe. Thanks, @JackCaoG!